### PR TITLE
Add a legacy features page, simplify the configuration

### LINKED ISSUE #2666: Document usage of cgxp for desktop in 2.0+
I noticed that there is some info in `CONST_CHANGELOG.txt` but I'm pretty sure that some infos are missing (checkers, database). Could you please add some doc about it?

Eg. I never had an answer to this point: https://github.com/camptocamp/c2cgeoportal/issues/2642#issuecomment-261463105

### PR DESCRIPTION
**Pull request by sbrunner**
fix #2666, #2665

### CODE REVIEW

File: doc/integrator/checker.rst [hunk 1]
```diff
@@ -26,45 +26,131 @@ The return code are:
...
-  use the ``phantomjs_routes`` configuration as an array of route name to check.
+The checker use the following configuration structure:
 
-Configuration in ``vars_<project>.yaml``:
+.. code:: yaml
+
+   vars:
+       # Global configuration
+       ...
+       checher:
```
Comment (ger-benjamin) [inline doc/integrator/checker.rst]: checher ?

File: doc/integrator/legacy_cgxp.rst.mako [hunk 2]
```diff
@@ -3,6 +3,8 @@
 CGXP
 ====
 
+It seal possible to have CGXP interface(s) in parallel of the ngeo interface(s).
```
Comment (ger-benjamin) [inline doc/integrator/legacy_cgxp.rst.mako]: Ilt seal -> it still ?

File: doc/integrator/legacy_cgxp.rst.mako [hunk 3]
```diff
@@ -20,6 +22,102 @@ The style sheet file is: ``<package>/static/css/proj.css``
...
+         checker:
+            ...
+            defaults:
+                ...
+                lang_files: [cgxp]
+     update_paths:
+     ...
+     - checker.defaults.lang_files
+
+Do not have ``check_collector.disabled`` in the ``update_paths`` (and don't readd it during
```
Comment (ger-benjamin) [inline doc/integrator/legacy_cgxp.rst.mako]: readd

File: doc/integrator/legacy_cgxp.rst.mako [hunk 4]
```diff
@@ -20,6 +22,102 @@ The style sheet file is: ``<package>/static/css/proj.css``
...
+the next upgrade steps).
+
+Update the interface in your ``<package>/__init__.py`` file:
+
+.. code:: python
+
+  add_interface(config, "<interface>", INTERFACE_TYPE_CGXP)
+
+
+The OGC proxy is deprecated because with modern browser we do not required to have it.
```
Comment (ger-benjamin) [inline doc/integrator/legacy_cgxp.rst.mako]: we do not required to have it... ->  It is not required to have it

File: doc/integrator/legacy_print.rst [hunk 5]
```diff
@@ -0,0 +1,79 @@
+.. _integrator_legacy_print:
+
+Print version 2.x
+=================
+
+The print version 2 cannot be used in the ngeo application be he steel can be used in the ngeo application.
```
Comment (ger-benjamin) [inline doc/integrator/legacy_print.rst]: be he steel -> but it still ?

File: doc/integrator/legacy_print.rst [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+
+The print version 3.x and the version 2.x cannot be used together.
+
+To use it you should::
+
+  * Keep the print war in your repository.
+  * Add ``PRINT_VERSION ?= 2`` in your project makefile.
+  * Add in the ``vars`` of your project vars file: ``print_url: http://localhost:8080/print-{instanceid}/pdf``.
+  * Desable the print v3 checker and enable the print v2 one, edit your project vars file:
```
Comment (ger-benjamin) [inline doc/integrator/legacy_print.rst]: Disable

File: doc/integrator/legacy_print.rst [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+    * ``vars/checker/defaults/print_template``
+    * ``vars/checker/defaults/print_center_lon``
+    * ``vars/checker/defaults/print_center_lat``
+    * ``vars/checker/defaults/print_scale``
+
+  * Do not have ``printproxy_capabilities`` in ``vars/checker/defaults/routes_disable``.
+  * Do not have a route named ``printproxy_info`` in ``vars/checker/defaults/routes``.
+  * Do not have ``checker_pdf3`` in ``vars/check_collector/disabled``.
+  * Have ``checker_pdf`` in ``vars/check_collector/disabled`` (it's possible that you can completely remove it).
+  * Be sure that you tahe thin in your project vars file::
```
Comment (ger-benjamin) [inline doc/integrator/legacy_print.rst]: you tahe thin -> you do this thing ?

File: doc/integrator/ngeo.rst [hunk 8]
```diff
@@ -116,3 +116,93 @@ To print the layers in the high quality you you should add those ``Metadata``:
...
+``interface_name`` is the name specified in the ``interface`` table,
+also used to create the route path.
+
+``interface_type`` may be either ``INTERFACE_TYPE_CGXP``, ``INTERFACE_TYPE_NGEO`` or
+``INTERFACE_TYPE_NGEO_CATALOGUE``. Constants available in ``c2cgeoportal``.
+
+Database
+--------
+
+The administration interface gives access to an ``interface`` table listing the
```
Comment (ger-benjamin) [inline doc/integrator/ngeo.rst]: ...table listing... -> table that lists....

File: doc/integrator/ngeo.rst [hunk 9]
```diff
@@ -116,3 +116,93 @@ To print the layers in the high quality you you should add those ``Metadata``:
...
+The default interfaces are ``desktop`` add ``mobile``.
+
+Checker
+-------
+
+Enable the checker for the new interface.
+
+We suggest to add only the main checker in the ``defaults`` it's what's done by default.
+
+And in the ``all`` (``vars.checker.all``) check all the ngeo interface in standard and debug mode:
```
Comment (ger-benjamin) [inline doc/integrator/ngeo.rst]: And in the ``all`` what ? in with the ``all`` case ? (And with ``all``)

File: doc/integrator/checker.rst [hunk 10]
```diff
@@ -26,45 +26,131 @@ The return code are:
...
+           default:
+               # Default checkers configuration
+           <type>: # type of checker get from the query string
+               # Checker configuration
+
+.. note::
+
+    If some of the checked services require an authenticated user, the
+    ``forward_headers`` permit to forward ``Cookie`` or ``Authorisation`` headers
```
Comment (ger-benjamin) [inline doc/integrator/checker.rst]: permit -> permits (or allows)

File: doc/integrator/ngeo.rst [hunk 11]
```diff
@@ -116,3 +116,93 @@ To print the layers in the high quality you you should add those ``Metadata``:
 .. note::
 
    See also: :ref:`administrator_administrate_metadata`, :ref:`administrator_administrate_ogc_server`.
+
+.. _integrator_ngeo_add:
+
+Add a new interface
+-------------------
+
+Be sure that's you have all the required files:
```
Comment (ger-benjamin) [inline doc/integrator/ngeo.rst]: that's -> that ? (Or even )-> be sure you have

File: doc/integrator/legacy_cgxp.rst.mako (hunk 3 above)
Comment (sbrunner) [inline doc/integrator/legacy_cgxp.rst.mako]: ?

File: doc/integrator/ngeo.rst (hunk 9 above)
Comment (sbrunner) [inline doc/integrator/ngeo.rst]: type all ...

**Pull request merged**